src/online/service/github.rs

```diff
@@ -189,19 +190,30 @@ Configure it with the following:
             source.name
         );
 
-        let body = serde_json::to_vec(&serde_json::json!({
-            "organization": destination.namespace,
+        let mut body_json = json!({
             "name": destination.name,
             "default_branch_only": true,
```

Comment:
This seems like something which could be a neat feature flag (i.e. should forks only copy the default branch?)